conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -920,4 +939,80 @@ public Boolean visitUnknown(String unknownType) {
             }
         });
     }
+
+    private CollectionType getCollectionType(Type type) {
+        return type.accept(new DefaultTypeVisitor<>() {
+            @Override
+            public CollectionType visitList(ListType value) {
+                if (!options.nonNullCollections()) {
+                    return new CollectionType(ConjureCollectionType.LIST, DO_NOT_USE_NON_NULL_COLLECTION_FACTORY);
```

Comment:
not-non-null double negative is a bit tricky to read, perhaps `NULLABLE_COLLECTION_FACTORY`?

Reply:
👍